light_curves/code_src/heasarc_functions.py

```diff
@@ -56,22 +70,75 @@ def make_hist_error_radii(missioncat):
 
 
 def heasarc_get_lightcurves(sample_table, *, catalog_error_radii={"FERMIGTRIG": 1.0, "SAXGRBMGRB": 3.0}):
-    """Searches HEASARC archive for light curves from a specific list of mission catalogs
+    """
+    Search selected HEASARC gamma-ray mission catalogs for transient events
+    that spatially coincide with entries in the input sample_table.
+
+    This function performs a TAP upload of the source coordinates, queries
+    each HEASARC catalog for events with sufficiently small localization
+    error circles, and records an event “marker” in the returned light-curve
+    structure. HEASARC does not provide multi-epoch flux light curves, so
+    each event is represented as a single time-stamped point with a fixed
+    placeholder flux.
 
     Parameters
     ----------
-    sample_table : `~astropy.table.Table`
-        Table with the coordinates and journal reference labels of the sources
+    sample_table : astropy.table.Table
+        Table containing the source sample. The following columns must be present:
+            coord : astropy.coordinates.SkyCoord
+                Sky position of each source.
+            objectid : int
+                Unique identifier for each source in the sample.
+            label : str
+                Literature label for tracking source provenance.               
     catalog_error_radii : dict
-        Catalogs to query and their corresponding max error radii. Dictionary key must be one of the tables listed
-        here: https://astroquery.readthedocs.io/en/latest/heasarc/heasarc.html#getting-list-of-available-missions.
-        Value must be the maximum error radius to include in the returned catalog of objects (ie., we are not
-        interested in GRBs with a 90degree error radius because they will fit all of our objects).
+        Dictionary specifying which HEASARC mission catalogs to search and the
+        **maximum allowed error radius (degrees)** for each.
+
+        Keys must be valid HEASARC table names, for example:
+            {"FERMIGTRIG": 1.0, "SAXGRBMGRB": 3.0}
+
+        Values:
+            float (degrees)
+                Maximum localization error radius to accept from that catalog.
+                This filters out events with very large positional uncertainties.
```

Comment:
I'm not sure what the standard way to document dict key and value requirements is, but currently it reads a little strange with the Keys example being a complete dictionary. Also, is it helpful to keep the tip that gets deleted above about not using a large error radius because it can match many objects? Here's a try...

```suggestion
        Key : str
            A valid HEASARC table name, for example: "FERMIGTRIG" or "SAXGRBMGRB".
        Value : float (degrees)
            Maximum localization error radius to accept from that catalog.
            Recommended values are about 1.0 to 3.0.
            Very large values have the potential to match many target objects.
```